### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is for next major.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
         "twig/twig": "^2.12.1 || ^3.0"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle": "^6.7",
-        "doctrine/doctrine-bundle": "^2.3.2",
+        "dama/doctrine-test-bundle": "^7.1",
```
Comment (jordisala1991) [inline composer.json]: If we use ^7.1 this forces to use dbal 3. wdyt @VincentLanglet ?

We have some 4.x versións of Sonata still allowing dbal 2. should we drop it already or we wait?

Comment (VincentLanglet) [reply composer.json]: Maybe we should wait.

This major will allow people to use symfony 5. People might need to first bump some dependencies before bumping dbal.

Comment (jordisala1991) [inline composer.json]:
```suggestion
        "dama/doctrine-test-bundle": "^6.7",
```

File: composer.json [hunk 2]
```diff
@@ -42,8 +42,8 @@
...
+        "doctrine/doctrine-bundle": "^2.4.3",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "doctrine/doctrine-bundle": "^2.3.2",
```

File: composer.json (hunk 1 above)
Comment (jordisala1991) [reply composer.json]: Fine by me, then lets only update the psalm extension

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**